Dockerfile.assisted-service

```diff
@@ -19,8 +19,9 @@ FROM registry.svc.ci.openshift.org/openshift/release:golang-1.15 as builder
 
 # Bring in the go dependencies before anything else so we can take
 # advantage of caching these layers in future builds.
-COPY go.mod go.mod
-COPY go.sum go.sum
+COPY go.mod .
+COPY go.sum .
+COPY pkg/requestid .
```

Comment:
I don't think this is going to be sustainable when we have a lot more of these.

Reply:
Do we need to copy all the package?

Reply:
Only the submodules, otherwise the `go mod download` fails because it won't be able to find the local path specified in `go.mod`.

All of this block (up to the `go mod download`) is an optimization so we can remove it if this becomes difficult to maintain.
Alternatively maybe there's a way to ignore errors from `go mod download` so we will get all the remote modules, but not fail on the missing local ones (I asked @omertuc to look into this)